src/test/java/software/amazon/documentdb/jdbc/DocumentDbConnectionTest.java

```diff
@@ -490,89 +479,37 @@ void testMultiProcessConnections(final DocumentDbTestEnvironment environment) th
         }
         environment.start();
 
-        final String connectionString = DocumentDbConnectionPropertiesTest
-                .buildInternalSshTunnelConnectionString(environment);
-        final int maxWaitTimePerClient = 5;
-        final List<String> commandLine = getCommandLine(connectionString, maxWaitTimePerClient);
-        final List<Process> processes = startClientRunnerProcesses(commandLine);
-        final Instant timeoutTime = Instant.now().plus(Duration.ofSeconds(maxWaitTimePerClient * 3));
-        assertProcessesCompleteNormally(processes, timeoutTime);
-    }
-
-    private static List<String> getCommandLine(final String connectionString, final int maxWaitTimePerClient)
-            throws SQLException, URISyntaxException {
-        final int numberOfClientsPerProcess = 5;
-        // This class name is provided in text because I don't want to mark the class public.
-        // For testing purposes, the class needs to be in the 'main' distribution - not the 'test' distribution.
-        final String clientRunnerClassName = DocumentDbSshTunnelServer.class.getPackage().getName()
-                + "." + "DocumentDbSshTunnelTestClientRunner";
-        return DocumentDbSshTunnelServer.getJavaCommand(
-                clientRunnerClassName,
-                connectionString,
-                String.valueOf(numberOfClientsPerProcess),
-                String.valueOf(maxWaitTimePerClient));
-    }
-
-    @SuppressFBWarnings("COMMAND_INJECTION")
-    private static List<Process> startClientRunnerProcesses(final List<String> commandLine) throws IOException {
-        final List<Process> processes = new ArrayList<>();
-        final int processCount = 5;
-        for (int i = 0; i < processCount; i++) {
-            processes.add(new ProcessBuilder(commandLine).start());
+        final int numberOfConnections = 20;
```

Comment:
To make the test bullet proof, would it make sense to try 100 connections?

Reply:
Set to 100.